common/reviews/api/ts-command-line.api.md

```diff
@@ -113,9 +113,9 @@ export abstract class CommandLineParameter {
     readonly environmentVariable: string | undefined;
     // @internal
     _getSupplementaryNotes(supplementaryNotes: string[]): void;
-    readonly groupName: string | undefined;
     abstract get kind(): CommandLineParameterKind;
     readonly longName: string;
+    readonly parameterGroup: string | typeof SCOPING_PARAMETER_GROUP | undefined;
```

Comment:
As long as you're okay with making this a part of the base API I'm okay with it!